src/__tests__/mockAPIGatewayV1Server.ts

```diff
@@ -29,8 +29,6 @@ function v1EventFromRequest(shouldBase64Encode: boolean) {
 
     // simplify the V1 event down to what our integration actually cares about
     const event: Partial<APIGatewayProxyEvent> = {
-      // @ts-expect-error (version actually can exist on v1 events, this seems to be a typing error)
```

Comment:
Why remove? This was added for correctness + to test functionality that was introduced.
https://github.com/apollo-server-integrations/apollo-server-integration-aws-lambda/pull/30

Reply:
That functionality is superseded by the fact that the end-user must define what event type is to be parsed. Also, I suspect the `version` key might not consistently exist. Nowhere in the AWS documentation does that `"version": "1"` exist. [Even in their own sample events](https://docs.aws.amazon.com/lambda/latest/dg/services-apigateway.html#apigateway-example-event).

My only guess is that the `version: 1` was added to the V1 event after the implementation of V2 was released. Now that our parser doesn't _need_ that key, I think its best to follow what the documentation and event type say, even if they are wrong. I figure we probably shouldn't test against an undocumented `version` key now that we don't have to.